### ISSUE TITLE
Add tests for download_dataset() function

### ISSUE THREAD

**Issue by damianooldoni** [inbo/etn#133]

**Comment by damianooldoni** [comment]
Using `2011_rivierprik` to test `download_dataset()` is still taking too much time. This is due to the third step, downloading `detections.csv` of size 22.9 MB. There are two options;

1. a even smaller project
2. introducing the option of downloading a subset of data files from the project, with download ALL as default option (something like `which = NULL` for the default behavior and `which = c("animals", "tags", "receivers")` for a subset of data)

Option 2 would make things more flexible without a tremendous increase of complexity. What do you think, @peterdesmet?

**Comment by peterdesmet** [comment]
1. 2015_homarus is a very small project (2 animals)
2. Sound like a good idea, but although you might not need to download detections, you still need to query them to know which receiver projects are included, so you're not going to save on time.

**Comment by damianooldoni** [comment]
1. Indeed, way smaller. But it takes still few seconds (~2.65s on my home connection). I think I will still opt for stubbing :hocho: Something like:
```r
detections_homarus <- read_csv("./tests/testthat/.../detections_stubbed.csv")
stub(download_dataset, "get_detections", detections_homarus)
```
With such tubbing will take less than 2s

2. If you think it can be an useful, although not urgent enhancement, I would refer to this in a new issue and I will add myself as assignee, ok?

**Comment by damianooldoni** [comment]
@peterdesmet : what about point 2? This would solve also the problem of adding detections (as csv or zipped) for tests as we can simply avoid to tests for detections. See your comment: https://github.com/inbo/etn/pull/142#pullrequestreview-535551403

**Comment by peterdesmet** [comment]
@damianooldoni we did avoid using stubbing right? And this issue is now implemented in #142?

**Comment by damianooldoni** [comment]
Yes. I close this issue.

Adding the option of downloading a subset of data files from the project can be seen as a new issue, if you still think it is a good idea to add such functionality to `download_dataset()`.
